Makefile

```diff
@@ -3,7 +3,7 @@
 all: speakeasy
 
 speakeasy: check-speakeasy
-	speakeasy run --skip-versioning --output console
+	speakeasy run --skip-versioning --output console --minimal
```

Comment:
Just checking if this change to Makefile is intentional

Reply:
Yes, this is intentional